rotkehlchen/tests/unit/test_assets.py

```diff
@@ -633,3 +638,43 @@ def test_symbol_or_name(database):
     assert Asset('xyz').symbol_or_name() == 'custom name'
     with pytest.raises(UnknownAsset):
         Asset('i-dont-exist').symbol_or_name()
+
+
+def test_load_from_packaged_db(globaldb: GlobalDBHandler):
+    """
+    Test that when resolving assets from the packaged db,
+    there is no attempts to write into the pacakged db.
+    """
+    packaged_db_path = Path(__file__).resolve().parent.parent.parent / 'data' / 'global.db'
+    backup = shutil.copy(packaged_db_path, packaged_db_path.with_suffix('.backup'))
```

Comment:
Why do this in the packaged db path? Create a temp directoy for this. Then you won't care about deletion.